tests/sentry/ingest/ingest_consumer/test_dlq.py

```diff
@@ -66,14 +66,14 @@ def test_dlq_invalid_messages(factories, topic_name, consumer_type) -> None:
     )
     strategy = factory.create_with_partitions(Mock(), Mock())
 
-    # Valid payload raises original error
-    with pytest.raises(Exception) as exc_info:
-        message = make_message(valid_payload, partition, offset)
-        strategy.submit(message)
-    assert not isinstance(exc_info.value, InvalidMessage)
+    # Valid payload with empty event raises original error (except Attachments, which filters empty events)
+    if consumer_type != ConsumerType.Attachments:
```

Comment:
Not sure why, but using this decorator causes attachments to skip processing for this test case. It might be the FilterCase.

Reply:
PR looks good, but this change makes me skeptical. If the decorator is just for convenience, why would it suddenly change behavior for attachments?

Reply:
I'll look into it some more. If it gets too complex/confusing I'll close the PR for now..

Reply:
Also see convo here https://github.com/getsentry/sentry/pull/67668#discussion_r1546915085